docs/synapse/devopsguide.rst

```diff
@@ -924,8 +924,466 @@ Kubernetes
 A popular option for Orchestration is Kubernetes. Kubernetes is an open-source system for automating the deployment,
 scaling and management of containerized applications. Synapse does work in Kubernetes environments.
 
-We are in the process of updating our Kubernetes related documentation. If you need assistance with deploying to
-a Kubernetes environment, please reach out to us directly via :ref:`synapse-support`.
+.. note::
+
+    If you are using these examples to get started with Synapse on Kubernetes, you may need to adapt them to meet
+    operational needs for your environment.
+
+.. _orch-kubernetes-deployment:
+
+Example Deployment
+~~~~~~~~~~~~~~~~~~
+
+The following deployment walks through deploying an example Synapse deployment ( based on :ref:`deploymentguide` ), but
```

Comment:
Maybe change this to "The following example"? Feels like it says deployment a lot of times in a row